db/packet_ldap.rb

```diff
@@ -5,6 +5,7 @@
 LDAP_EVAL_BASE   = 'cn=Evaulations,ou=Committees,dc=csh,dc=rit,dc=edu'.freeze
 LDAP_EBOARD_CN   = 'eboard'.freeze
 LDAP_INTRO_CN    = 'intromembers'.freeze 
+LDAP_FAILED_CN    = 'failed'.freeze 
```

Comment:
What's an example for when this is necessary?

Reply:
If someone who passed their Introductory Evaluations process gets onto the housing board. Then they proceed to fail Membership Evaluations. If they stay on the roster for living on floor the next year they will have a disabled account (with the roomNumber field updated) but would have no authority to participate in the packet process as an upperclassman.